### PR TITLE
Add description of Plot Scripts

### PR DESCRIPTION
**Pull request by 200km**
## Overview
Add description of Plot Scripts

## Issue
- #37 

## Details
Add description of Plot Scripts

##  Validation results
See the new document.

## Scope of influence
NA

## Supplement
NA

## Note
NA

### CODE REVIEW

File: General/HowToVisualizeSimulationResults.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# How to Visualize Simulation Results
+
+## 1.  Overview
+The S2E generates the CSV format log file as the simulation results at the `data/log` directory. Users can use them to check the result of thier simulation with the following methods.  
+- Open the CSV file with a spreadsheet application like Excel, and make charts to analyze and/or visualize the results.
+- Open the CSV file with user's handmaded programs to analyze and/or visualize the resutls.
+- Use the `Plot Scripts` provided by the `S2E-CORE` to visualize the results.
+
+We are trying to increase the `Plot Scripts` to share the analysis method with many users and to improve the usability of S2E. We are welcome to ad d new `Plot Scripts` by users.
```
Comment (seki-hiro) [inline General/HowToVisualizeSimulationResults.md:9]: [NITS] ad d -> add

File: General/HowToVisualizeSimulationResults.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  - By using the `Pipenv`, users can easily construct the virtual environment to execute the scripts provided by the S2E
+- Example of the Visualization
+  - Satellite trajectory on the earth map and ground station visibility analysis
+    - The red star means the ground station.
+    - The blue line shows the satellite trajectory and it turns red when the satellite is visible from the ground station.
+    <div align="center">
+      <img src="./figs/plot_gs_visibility.JPG" alt="plot_gs_visibility" style="zoom:50%;" />
+    </div>
+  - Direction information at the satellite body fixed frame
+    - Users can see the direction vector of the sun, earth, moon, and the velocity of the satellite at the body fixed frame to checkt the attitude control works well.
```
Comment (seki-hiro) [inline General/HowToVisualizeSimulationResults.md:28]: [NITS] checkt -> check

**Review by seki-hiro — APPROVED** [review]
細かいコメントしましたが，Approveしています．

File: General/HowToVisualizeSimulationResults.md (hunk 2 above)
Comment (200km) [reply General/HowToVisualizeSimulationResults.md]: Thanks. I fixed the typos.

**Pull request merged**